src/Sensors/Sentinel1/FileIO/Metadata/Sentinel1Metadata.jl

```diff
@@ -121,7 +130,7 @@ Sentinel1ImageInformation
     It takes a dictionary containing the full sentinel-1 swath metadata and extracts the Sentinel1ImageInformation as a structure. Input in the Sentinel1ImageInformation file:
         range_pixel_spacing: Pixel spacing between range samples [m].
         azimuth_frequency: Azimuth line frequency of the output image [Hz]. This is the inverse of the azimuth_timeInterval.
-        slant_range_time: Two-way slant range time to first sample [milli sec].
+        slant_range_time_seconds: Two-way slant range time to first sample [sec].
```

Comment:
I prefer [s] or [seconds] just to be clear.

Reply:
I will fix these